### ISSUE TITLE
The report system is bugged and abusable 

### ISSUE THREAD

**Issue by carpevinum101** [ValveSoftware/Dota2-Gameplay#6850]
### Description

Hi, 

I am your regular dota player that was 10k behaviour for like 7 years until I got a couple of abandons becuase russians kept bombing my city.  I got to 6k behaviour then it has been a year of not being able to get out.  I play 80 hours a week worth of turbos to try and offset it, commending everyone , doing my best to win while people in parties ruin games on purpose, but I have to commend them in hopes of getting normal queue times and normal dota one day. 

Recently I've been continuously targeted by parties doing absolutely nothing wrong, while people just ruin games, pick 2nd carry in my lane even though I picked first, or just straight up reporting me if I support.  

This happened. I am under 3k behaviour score with 24h ban , while I got game ruined by parties who disband before reporting so the reports count as separate. 

https://imgur.com/a/pHKU0IQ

7000361744
7000407150

**Comment by BoneGuardSkuLL** [comment]
Yeah today i have noticed ppl simply report any russian players for war reasons and its very sad. The report system is highly abusable
